<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>845</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Part Inventory Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Nbr_OF_Holes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1091</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inventory Details</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Search of reference with count level lower or equal to</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="Table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1081</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Century Gothic&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Parts Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Min Area</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max Area</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nbr Holes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Min Diameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max Diameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="ref_filter_box">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>7</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_btn">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>610</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Referesh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Inventory Statsitics</string>
     </attribute>
     <widget class="QLabel" name="lbl_ref_num">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Number Of Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_part_num">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Number Of Part Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_holes">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Min.Number Of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_hole">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Max.Number Of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_chnge_ref">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Century Gothic&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_chnge_part">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Century Gothic&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_chnge_min_hole">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>40</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Century Gothic&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_chnge_max_hole">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>190</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Century Gothic&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_hole_ref_1">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>40</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Century Gothic&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_hole_ref_2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>190</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Century Gothic&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Top three reference with minimum inventory level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="check_btn">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>370</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 202, 202);</string>
      </property>
      <property name="text">
       <string>CHECK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_btn_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>590</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 202, 202);</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>851</width>
        <height>221</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Edit Inventory</string>
     </attribute>
     <widget class="QLabel" name="Ref_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="PartName_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Part Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="Min_Area_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Min Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="MAx_Area_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Max Area</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ref_Input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Part_Name_Input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Min_Area_Input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Max_Area_Input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nbr_of_holes_Input">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Min_diameter_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>190</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Max Diameter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Max_Diameter_Input">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Min_Diameter_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>150</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Min Diameter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Min_Diameter_Input">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Nbr_Of_holes_lbl">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>110</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Number Of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="Count_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="ID_LABEL">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>330</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>330</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>update</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Count_SPinBOx_input">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="ID_Input_1">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 30pt &quot;Century Gothic&quot;;
color:rgb(0, 255, 255);
background-color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string>Parts Inventory Manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
